apps/transport/lib/transport_web/live/backoffice/proxy_config_live.ex

```diff
@@ -77,9 +113,20 @@ defmodule TransportWeb.Backoffice.ProxyConfigLive do
         ttl: resource.ttl
       }
       |> add_cache_state()
+      |> add_stats(stats)
     end)
   end
 
+  defp add_stats(item, stats) do
+    metrics_target = Unlock.Controller.Telemetry.target_for_identifier(item.unique_slug)
+    counts = stats[metrics_target] || %{}
+
+    Map.merge(item, %{
+      stats_external_requests: Map.get(counts, "proxy:request:external", 0),
+      stats_internal_requests: Map.get(counts, "proxy:request:internal", 0)
```

Comment:
Il y a moyen de deviner les chaînes de caractère ici ? Ça me parait compliqué à faire. Bon après c'est sous tests.

Reply:
Deviner non, toutefois j'ai pu DRYer correctement ici:

https://github.com/etalab/transport-site/compare/220c0b954607bce1c89b7b7e55dd399ff417f49f...1feff5c

C'est un peu "over the top" toutefois je pense que ça aura des vertus pour la suite : il est plus facile de détecter une erreur de compilation ou de match, qu'un filtre raté ici.

Merci pour la remarque, je te laisse vérifier que ça te va et résoudre la conversation ?